Project.toml

```
@@ -35,17 +35,18 @@ Tracker = "9f7883ad-71c0-57eb-9f7f-b5c9e6d3789c"
 AbstractMCMC = "0.5.2"
 AdvancedHMC = "0.2.20"
 AdvancedMH = "0.4"
-Bijectors = "0.4.0, 0.5, 0.6"
+Bijectors = "0.6.2"
 BinaryProvider = "0.5.6"
-Distributions = "0.21.11, 0.22"
-DistributionsAD = "0.4.1"
+Distributions = "0.22"
+DistributionsAD = "0.4.3"
 DocStringExtensions = "0.8"
 DynamicPPL = "0.4"
 EllipticalSliceSampling = "0.2"
+FiniteDifferences = "0.9"
 ForwardDiff = "0.10.3"
 Libtask = "0.3.1"
 LogDensityProblems = "^0.9, 0.10"
-MCMCChains = "3"
+MCMCChains = "3.0"
```

This change does not restrict the compatible versions of MCMCChains, so maybe just keep
```suggestion
MCMCChains = "3"
```
to avoid any confusion (some people might assume incorrectly that it is compatible with only MCMCChains [3.0.0, 3.1.0) whereas actually it is still compatible with MCMCChains [3.0.0, 4.0.0)?